glob.go

```diff
@@ -74,20 +75,34 @@ func QuoteMeta(pattern string) string {
 
 // toNixPath converts the path to the nix style path
 // Windows style path separators are escape characters so cause issues with the compiled glob.
-func toNixPath(path string) string {
-	return filepath.ToSlash(filepath.Clean(path))
+func toNixPath(s string) string {
+	return path.Clean(filepath.ToSlash(s))
 }
 
 // Glob returns all files that match the given pattern in the current directory.
+// If the given pattern indicates an absolute path, it will glob from `/`.
 func Glob(pattern string, opts ...OptFunc) ([]string, error) {
+	prefix := "./"
+	if strings.HasPrefix(pattern, stringSeparator) {
+		prefix = stringSeparator
+		opts = append(opts, func(opts *globOptions) {
+			opts.prefix = prefix
+		}, WithFs(os.DirFS(prefix)))
```

Comment:
If I understand this correctly `globOptions.fs` will be a `os.DirFS` no matter what the user provides via `WithFS`, right?

Reply:
yeah, I think this is wrong actually... 🤔